src/Common/src/System/Dynamic/Utils/TypeUtils.cs

```diff
@@ -41,13 +41,9 @@ public static void ValidateType(Type type)
             if (type != typeof(void))
             {
                 // A check to avoid a bunch of reflection (currently not supported) during cctor
```

Comment:
I couldn't tell what it was referring to. I guessed the void test since the rest of the checks certainly aren't to avoid reflection. That reflection isn't available during cctors was certainly news to me, and led me to wonder if perhaps it got called into in some cctor that was related to reflection itself, but I didn't see anything that seemed to fit that bill.

On the basis that what you don't understand you should leave alone, I hence left it alone, though it would be slightly nicer to make this a single `if` with a `&&`.